tests/internal/sqs.js

```diff
@@ -26,4 +26,16 @@ export function sqsTestSuite(data) {
         // Assert
         expect(response.messageId).to.be.a('string');
     })
+
+    describe('send FIFO message', () => {
+        // Act
+        const response = sqsClient.sendMessage({
+            queueUrl, messageBody: 'test',
+            messageGroupId: 'testGroupId',
+            messageDeduplicationId: 'testDeduplicationId'
+        });
+
+        // Assert
+        expect(response.messageId).to.be.a('string');
```

Comment:
Would it be relevant to add some expect statements dedicated to the `messageDeduplicationId` and `messageGroupId` here, for completeness? I'm not too familiar with SQS, so there could be an obvious reason or a subtlety that I missing though 🙇🏻

Reply:
The fundamental reason why is that the tests don't have any mock infrastructure now. Ideally, I would have preferred to make the call with a spy and then confirm that the sqsClient called `http` with the expected body. I did not want to presume to add any additional dependencies to provide this mocking.

Reply:
Thanks for your explanation. I've filled the gap of my lack of knowledge on the matter and realized that SQS doesn't return these attributes in its response. Let's keep it at that 👍🏻